ReflectometryServer/parameters.py

```diff
@@ -860,7 +916,8 @@ def _initialise_sp_from_file(self):
 
     def _initialise_sp_from_motor(self, _):
         STATUS_MANAGER.update_error_log("Optional parameter, {}, was asked up init from motor".format(self.name))
-        STATUS_MANAGER.update_active_problems("Optional Parameter updating from motor")
+        STATUS_MANAGER.update_active_problems(ProblemInfo("Optional Parameter updating from motor", self.name,
+                                                          Severity.MAJOR_ALARM))
```

Comment:
Is this really major? Seems informational?

Reply:
Have added comment and will add note for wiring tables to snags list because we need user input